core/services/job/job_orm_test.go

```diff
@@ -849,6 +851,57 @@ func Test_FindPipelineRunIDsByJobID(t *testing.T) {
 
 		assert.Equal(t, run.ID, runIDs[0])
 	})
+
+	jobs := make([]job.Job, 10)
+	for j := 0; j < len(jobs); j++ {
+		_, bridge := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+		_, bridge2 := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+		jobID := uuid.NewV4().String()
+		key, err := ethkey.NewV2()
+
+		require.NoError(t, err)
+		jb, err = ocr.ValidatedOracleSpecToml(cc,
+			testspecs.GenerateOCRSpec(testspecs.OCRSpecParams{
+				JobID:              jobID,
+				Name:               fmt.Sprintf("Job #%v", jobID),
```

Comment:
Is there a unique index on this?

Reply:
I tried everything I could think of to re-use the same objects, but everything gave me lots of uniqueness constraint errors... the rules are pretty strict.